server/src/daemon/service.rs

```diff
@@ -52,7 +52,7 @@ impl Service {
 
     #[zbus(out_args("collection", "prompt"))]
     pub async fn create_collection(
-        &self,
+        &mut self,
```

Comment:
This shouldn't become mut, instead make the collections field a mutex

Reply:
done, thanks!